.travis.yml

```diff
@@ -9,7 +9,9 @@ services:
 before_install:
   - sudo apt-get -qq update
 install:
-  - pip install molecule
+  - pip install molecule==2.22rc1 
+  - pip install flake8
+  - pip install cerberus==1.3.1
```

Comment:
these 2 dependencies were required due to incomplete deps in molecule `2.22rc1`
Can be removed again in `2.22rc3`